test.jl

```diff
@@ -19,6 +20,11 @@ initial_state = (nAge=9, nHS=8, nIso=4, nI=4, nTest=4)
 	trFunc_HospitalAdmission(;initial_state...)
 	@test py"np.transpose(trFunc_HospitalDischarge())"≈
 	trFunc_HospitalDischarge(;initial_state...)
+	@test py"np.transpose(trFunc_travelInfectionRate_ageAdjusted(10))"≈
+	trFunc_travelInfectionRate_ageAdjusted(t)
+end
+
+@testset "not working"
```

Comment:
There is some issue with relative path as we make pkg , things will get resoleved:)

Reply:
Okay